### PR TITLE
Added a function to create iterator with specific CF and ReadOptions

### PR DESCRIPTION
**Pull request by elichai**
Added after Discussion with @iSynaptic https://github.com/rust-rocksdb/rust-rocksdb/issues/273

### CODE REVIEW

File: src/db.rs [hunk 1]
```diff
@@ -1093,6 +1093,15 @@ impl DB {
         DBIterator::new(self, &readopts, mode)
     }
 
+    pub fn iterator_cf_opt(
```
Comment (aleksuss) [inline src/db.rs]: @elichai It would be nice to add some description for public method

Comment (elichai) [reply src/db.rs]: Done.

**Comment by elichai** [comment]
Thanks!
@iSynaptic If you could publish a new version it would be great :)

**Comment by iSynaptic** [comment]
See #287

**Comment by iSynaptic** [comment]
Just released 0.12.1.

**Pull request merged**